src/components/UserAccess/__tests__/UserAccessListView.spec.tsx

```diff
@@ -1,16 +1,36 @@
-import * as React from 'react';
 import { Table as PfTable, TableHeader } from '@patternfly/react-table/deprecated';
-import { act, fireEvent, screen } from '@testing-library/react';
-import { SpaceBindingRequest } from '../../../types';
-import { createK8sWatchResourceMock, namespaceRenderer } from '../../../utils/test-utils';
-import { SBRListRow } from '../SBRListRow';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { defaultKonfluxRoleMap } from '../../../__data__/role-data';
+import { mockRoleBinding, mockRoleBindings } from '../../../__data__/rolebinding-data';
+import { useRoleMap } from '../../../hooks/useRole';
+import { useRoleBindings } from '../../../hooks/useRoleBindings';
+import { useSearchParam } from '../../../hooks/useSearchParam';
+import { useAccessReviewForModel } from '../../../utils/rbac';
+import { useWorkspaceInfo } from '../../Workspace/useWorkspaceInfo';
+import { RBListRow } from '../RBListRow';
 import { UserAccessListView } from '../UserAccessListView';
 
 jest.mock('react-router-dom', () => ({
   Link: (props) => <a href={props.to}>{props.children}</a>,
-  useSearchParams: () => React.useState(() => new URLSearchParams()),
+  useParams: jest.fn(),
+  useNavigate: jest.fn().mockReturnValue(jest.fn()),
 }));
 
+jest.mock('../../Workspace/useWorkspaceInfo', () => ({
+  useWorkspaceInfo: jest.fn(),
+}));
+
```

Comment:
you shouldn't need to use workspaces hook anymore. If you are using useWorkspaceInfo hook in code somewhere, Please replace it with useNamespace

Reply:
Fixed. I gave up all useWorkspaceInfo in my patch.